sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -43,7 +43,7 @@ class StateOperatorProgress private[sql](
     val numRowsTotal: Long,
     val numRowsUpdated: Long,
     val memoryUsedBytes: Long,
-    val numLateInputs: Long,
+    val numDroppedRowsByWatermark: Long,
```

Comment:
nit. Just a question. For the variable naming, `numRowsDroppedByWatermark` seems more consistent with `numRowsTotal` and `numRowsUpdates` within this `StateOperatorProgress` class, doesn't it? (cc @zsxwing since this acmes from his comment.)

Reply:
Yeah that sounds better in consistency (numRows + blabla explaining how the number is measured) and provides same meaning. Thanks for the suggestion.